docs/3.0/develop/logging.mdx

```diff
@@ -336,3 +336,17 @@ and SciPy logging statements with your flow and task run logs, use:
 
 Configure this setting as an environment variable or in a profile. See
 [Settings](/3.0/manage/settings-and-profiles/) for more details about how to use settings.
+
+## Access logs from the command line
+
+You can retrieve logs for a specific flow run ID using Prefect's CLI:
+
+```bash
+prefect flow-run logs $ID
```

Comment:
can we be a bit more explicit with this `$ID`? like `<flow run ID>` or `$FLOW_RUN_ID` or even explain that you could set it? it just seems sort of random as is

Reply:
@EmilRex applied my suggestion to make it more descriptive and remove the $. Marking resolved.